module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -103,6 +103,13 @@ class KohaRest extends \VuFind\ILS\Driver\AbstractBase implements
      */
     protected $defaultPickUpLocation;
 
+    /**
+     * Whether to allow canceling holds in transit. Default is false.
+     *
+     * @var bool
+     */
+    protected $allowCancelInTransit;
```

Comment:
Probably wouldn't hurt to initialize this to false to ensure it's not unset if init() is somehow bypassed (unlikely, but maybe possible in testing, etc.).

Reply:
Done. I added another one as well while at it.